tests/test-distribution/test-distribution-common/src/test/java/org/eclipse/jetty/tests/distribution/StatsTests.java

```diff
@@ -56,19 +42,19 @@ public void testStatsServlet(String env) throws Exception
         String[] args1 = {
             "--create-startd",
             "--approve-all-licenses",
-            "--add-modules=resources,server,http,stats," + toEnvironment("webapp", env) + "," + toEnvironment("deploy", env)
+            "--add-modules=resources,server,http,statistics," + toEnvironment("webapp", env) + "," + toEnvironment("deploy", env)
         };
         try (JettyHomeTester.Run run1 = distribution.start(args1))
         {
             assertTrue(run1.awaitFor(START_TIMEOUT, TimeUnit.SECONDS));
+            run1.getLogs().forEach(System.err::println);
             assertEquals(0, run1.getExitValue());
 
+            // Make a context
             Path webappsDir = distribution.getJettyBase().resolve("webapps");
             FS.ensureDirExists(webappsDir.resolve("demo"));
-            FS.ensureDirExists(webappsDir.resolve("demo/WEB-INF"));
 
-            distribution.installBaseResource("stats-webapp-" + env + "/index.html", "webapps/demo/index.html");
-            distribution.installBaseResource("stats-webapp-" + env + "/WEB-INF/web.xml", "webapps/demo/WEB-INF/web.xml");
+            // TODO add some actual content to it
```

Comment:
I think this TODO can go away, the `StatisticsHandler` is installed as the root of the handler tree and there already is a deployed webapp that can be queried to check the stats.

Reply:
Currently it is just an empty directory, there is not webapp deployed.